pyproject.toml

```
@@ -0,0 +1,93 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "packitos"
+authors = [
+    { name = "Red Hat", email = "user-cont-team@redhat.com" },
+]
+description = "A set of tools to integrate upstream open source projects into Fedora operating system."
+readme = "README.md"
+license = { file = "LICENSE" }
+requires-python = ">=3.9"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Software Development",
+    "Topic :: Utilities",
+]
+dynamic = ["version"]
+keywords = [
+    "git",
+    "packaging",
+    "fedora",
+    "rpm",
+    "dist-git",
+]
+dependencies = [
+    "GitPython",
+    "PyYAML",
+    "cccolutils",
+    "click",
+    "copr",
+    "lazy_object_proxy",
+    "marshmallow",
+    "marshmallow-enum",
+    "munch",
+    "ogr",
+    "packaging",
+    "python-gnupg",
+    "requests",
+    "requests-kerberos",
+    "setuptools",
+    "specfile",
+    "tabulate",
+    "bodhi-client >= 7.0.0",
+    "koji",
+    "rpkg",
+    "cachetools",
+    "python-fedora",
+]
+
+[project.urls]
+Homepage = "https://github.com/packit/packit"
+
+[project.optional-dependencies]
+testing = [
+    "pytest",
+    "flexmock",
+    "deepdiff",
+    "distro",
+]
+dev = [
+    "packitos[testing]",
+    "pre-commit",
+]
+
+[project.scripts]
+packit = "packit.cli.packit_base:packit_base"
+
+[tool.hatch]
+build.targets.wheel.only-include = ["packit"]
```

```suggestion
build.targets.wheel.packages = ["packit"]
```

should fix the EPEL 9 failures